src/sentry/models/release.py

```diff
@@ -675,14 +675,6 @@ def add_dist(self, name, date_added=None):
             defaults={"date_added": date_added, "organization_id": self.organization_id},
         )[0]
 
-    def get_dist(self, name):
```

Comment:
I removed this function because it is not used anywhere else, and its callers are at risk of confusing "does not exist" with `dist=None` i.e. the default distribution.